cli/embedded-app/src/views/IframeEmbed.vue

```diff
@@ -10,6 +10,7 @@
       id="frameRouter"
       v-bind:route="frameRoute"
       v-on:toastRequest="displayToast"
+      v-on:keyDown="handleKeyEvent"
```

Comment:
keydown as an event name seems a little off to me since it could be confused with normal key down.  I think it might be nice to tie it to the registeredKeys you used to subscribe to them.

Reply:
As in the name should have some mention of registeredKeys or something like that? For example registeredKeyFired